src/main/java/com/itacademy/cms/model/Category.java

```diff
@@ -20,9 +18,7 @@
 @Setter
 public class Category extends BaseEntity {
 
-  @OneToMany(cascade = CascadeType.ALL)//mappedBy = "category", fetch = FetchType.LAZY)
+  @OneToMany(mappedBy = "category", fetch = FetchType.LAZY)
```

Comment:
If we use mappedBy with fetch, at the endpoint category will recursively call course and RunnOutOfMemmory

Reply:
Ivan you also can do, i am not in charge of Category endpoint I am just trying to put all together and make it work